cinema/models.py

```diff
@@ -102,6 +107,14 @@ def clean(self):
                     }
                 )
 
+    def clean(self):
+        Ticket.validate_ticket(
+            row=self.row,
+            seat=self.seat,
+            movie_session=self.movie_session,
+            error_to_raise=ValueError
```

Comment:
In the `clean` method of the `Ticket` model, `ValueError` is used as the `error_to_raise` parameter. However, for validation errors in Django, you should use `ValidationError` instead of `ValueError`. This ensures that the error is properly handled by Django's validation system.